client/my-sites/plugins/controller.js

```diff
@@ -111,9 +111,11 @@ function renderPluginList( context, basePath ) {
 			: ''
 		);
 
-	analytics
-	.pageView
-	.record( context.pathname.replace( site.domain, ':site' ), analyticsPageTitle );
+	if ( site ) {
+		analytics
```

Comment:
From the controller, we can also do:
```
context.store.dispatch( recordPageView( url, title, service ) );
```
Or we can render the `<PageViewTracker />` component in the PluginsListComponent